packages/rxjs/integration/import/fixtures/browser/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>RxJS Import Integration Test</title>
+  </head>
+  <body>
+    <script type="module" src="./browser-test.js"></script>
```

Comment:
I'll be honest, this is the first "real" `<script type="module">` I've ever written.